spec/std/file_spec.cr

```diff
@@ -558,102 +558,14 @@ describe "File" do
-      File.expand_path("../bin", "tmp/x").should eq(File.join([base, "tmp", "bin"]))
-      File.expand_path("../bin", "x/../tmp").should eq(File.join([base, "bin"]))
-    end
-
-    pending_win32 "expand_path for common unix path gives a full path" do
-      File.expand_path("/tmp/").should eq("/tmp/")
-      File.expand_path("/tmp/../../../tmp").should eq("/tmp")
-      File.expand_path("").should eq(base)
-      File.expand_path("./////").should eq(File.join(base, ""))
-      File.expand_path(".").should eq(base)
-      File.expand_path(base).should eq(base)
-    end
-
-    pending_win32 "converts a pathname to an absolute pathname, using ~ (home) as base" do
-      File.expand_path("~/", home: true).should eq(File.join(home, ""))
-      File.expand_path("~/..badfilename", home: true).should eq(File.join(home, "..badfilename"))
-      File.expand_path("..", home: true).should eq("/#{base.split('/')[0...-1].join('/')}".gsub(%r{\A//}, "/"))
-      File.expand_path("~/a", "~/b", home: true).should eq(File.join(home, "a"))
-      File.expand_path("~", home: true).should eq(home)
-      File.expand_path("~", "/tmp/gumby/ddd", home: true).should eq(home)
-      File.expand_path("~/a", "/tmp/gumby/ddd", home: true).should eq(File.join([home, "a"]))
-    end
-
-    pending_win32 "converts a pathname to an absolute pathname, using ~ (home) as base (trailing /)" do
-      prev_home = home
-      begin
-        ENV["HOME"] = File.expand_path(datapath)
-        File.expand_path("~/", home: true).should eq(File.join(home, ""))
-        File.expand_path("~/..badfilename", home: true).should eq(File.join(home, "..badfilename"))
-        File.expand_path("..", home: true).should eq("/#{base.split('/')[0...-1].join('/')}".gsub(%r{\A//}, "/"))
-        File.expand_path("~/a", "~/b", home: true).should eq(File.join(home, "a"))
-        File.expand_path("~", home: true).should eq(home)
-        File.expand_path("~", "/tmp/gumby/ddd", home: true).should eq(home)
-        File.expand_path("~/a", "/tmp/gumby/ddd", home: true).should eq(File.join([home, "a"]))
-      ensure
-        ENV["HOME"] = prev_home
-      end
-    end
-
-    pending_win32 "converts a pathname to an absolute pathname, using ~ (home) as base (HOME=/)" do
-      prev_home = home
-      begin
-        ENV["HOME"] = "/"
-        File.expand_path("~/", home: true).should eq(home)
-        File.expand_path("~/..badfilename", home: true).should eq(File.join(home, "..badfilename"))
-        File.expand_path("..", home: true).should eq("/#{base.split('/')[0...-1].join('/')}".gsub(/\A\/\//, "/"))
-        File.expand_path("~/a", "~/b", home: true).should eq(File.join(home, "a"))
-        File.expand_path("~", home: true).should eq(home)
-        File.expand_path("~", "/tmp/gumby/ddd", home: true).should eq(home)
-        File.expand_path("~/a", "/tmp/gumby/ddd", home: true).should eq(File.join([home, "a"]))
-      ensure
-        ENV["HOME"] = prev_home
-      end
+  describe ".expand_path" do
```

Comment:
I would suggest to leave a reference to the specs for `Path#expand` here, like we do with `.join`.

```suggestion
  # There are more detailled specs for `Path#expand` in path_spec.cr
  describe ".expand_path" do
```